edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.DiagnosticsComponent/Uploader/FileWriter.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.Azure.Devices.Edge.Agent.DiagnosticsComponent
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// Writes to a local file instead of uploading.
```

Comment:
What is the scenario for this feature?  do we expect customers to use these metrics output file? are these metrics output file using prometheus format?

Reply:
This is mainly used for testing. Once iothub uploading is completed, I might remove it, but it is very useful for validation. It might be useful when you add metrics scraping to unit/long-haul tests.